packages/adminrouter/extra/src/includes/metrics-http-level.conf

```diff
@@ -0,0 +1,15 @@
+# Output fine-grained http directive level metrics using the VTS module.
+# 
+# Caution:
+# A request which matches a location that has a VTS module filter defined will be
+# counted on the Nginx server and location directive level filter as well.
+#
+# This only exists in order for the Nginx VTS module to calculate the average latency
+# per second over all requests for us. This could potentially be removed once we
+# calculate the average latency ourselves in Grafana.
+# 
+# By supplying no value to http= and all= the keys will only be used by our custom
+# Prometheus config in the dcos-monitoring package to relabel the metric __name__
```

Comment:
"our custom Prometheus config in the dcos-monitoring package" - is the relabelling always going to be there?

Reply:
Nope, fixed.